specification/network/resource-manager/Microsoft.Network/stable/2019-09-01/applicationGateway.json

```diff
@@ -1436,6 +1436,13 @@
           "type": "string",
           "description": "Host name of HTTP listener."
         },
+        "hostnames": {
```

Comment:
please add new properties at the end to avoid breaking changes in SDKs

Reply:
Updated.